CONTRIBUTING.md

```diff
@@ -90,6 +90,14 @@ and add the label_pipelines_appstudio_openshift_io_type to it.
 Signed-off-by: Avi Biton <abiton@redhat.com>
 ```
 
+### Pull Request Description
+The PR description (the first comment at the top of the PR) can usually be derived from
+the commit message body - especially if the PR consists of a single commit.
```

Comment:
Yeah, but if you change your commit message after the PR was already created, the PR description will not update and you'll have to update it manually....

Reply:
It's about the content rather than about the technicalities of how the content gets from the commit message to the pr description. It's up to the owner to fulfil this requirement. If GitHub helps to some extent with that, then alright.